### PR TITLE
Include multiple Safe deployment in the scripts

### PR DESCRIPTION
**Pull request by fedgiac**
Use the contract created in PR #113 to make bracket deployment in the scripts more efficient.

This PR has two prerequisites before merging:
1. PR #113 is finalized
2. The FleetFactory contract is deployed in at least mainnet and Rinkeby.

### CODE REVIEW

File: scripts/utils/trading_strategy_helpers.js [hunk 1]
```diff
@@ -186,16 +186,13 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
...
     const gnosisSafeMasterCopy = await gnosisSafeMasterCopyPromise
 
-    // TODO - Batch all of this in a single transaction
-    const createdSafes = []
-    for (let i = 0; i < fleetSize; i++) {
-      const newSafeAddress = await deploySafe(gnosisSafeMasterCopy, proxyFactory, [masterAddress], 1)
-      log("New Safe Created", newSafeAddress)
-      createdSafes.push(newSafeAddress)
-    }
+    const transcript = await fleetFactory.deployFleet(masterAddress, fleetSize, gnosisSafeMasterCopy.address)
```
Comment (josojo) [inline scripts/utils/trading_strategy_helpers.js]: really cool that it is soo easy

**Comment by josojo** [comment]
I was just quickly merging the conflicts so that I can see the diff correctly

File: test/test-utils.js [hunk 2]
```diff
@@ -22,8 +23,35 @@ const addCustomMintableTokenToExchange = async function(exchange, symbol, decima
     token: token,
   }
 }
+const deploySafe = async function(gnosisSafeMasterCopy, proxyFactory, owners, threshold) {
```
Comment (josojo) [inline test/test-utils.js:26]: Should we use here your new logic as well? 
using it in all our tests would give us more confidence.

Although, I am fine with using the old logic, as other tests are already in place for the new proxy deployments...

Comment (fedgiac) [reply test/test-utils.js:26]: The new logic is already used everywhere, this function should be only used to create the master safe now.

**Pull request merged**